### ISSUE TITLE
mail marked as read and archived still unread in Inbox of Gmail

### ISSUE THREAD

**Issue by puttin** [nylas/N1#2457]
##### Are there any related issues?

nope

##### What operating system are you using?
macOS 10.11.5

##### What version of N1 are you using?
0.4.45-7637265 (0.4.45-7637265)

--

**Bug?**
##### Do you have any third-party plugins installed?
nope

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
I use Gmail only

##### Is the issue reproducible with a particular attachment, message, signature, etc?

nope

--

**Feature Request?**
##### Does this feature exist in another mail client or tool you use?
nope

**Comment by jeffmcneill** [comment]
I've got the same issue:

OSX Version 10.10.05 Yosemite
Nylas Version 0.4.45-7637265 (0.4.45-7637265)
Mailbox is Google Apps for Domains

This is true with all clients, that is:

- Open nylas
- read a new email and/or archive/move/delete an email
- Open another email client
- Behavior: emails do not show being read, archived, moved or deleted
- Expected behavior: emails will indicate same status/location as in Nylas

This is new behavior with the new version. I will try and roll back, because this new behavior is not meeting my requirements. I can't go through all my mail and repeat all actions twice.

To reiterate:

- Mail is being sent from Nylas, but it is not showing up in Google Apps mailboxes either as inbound new, or as sent
- Basically, all mail actions are not being SYNCED. Sync engine appears to not be working from a Nylas --> Google Apps mailboxes direction.

**Comment by elchinot7** [comment]
Same issue in Debian 8. 
Nylas activity is not synced with Gmail apps.

**Comment by pfista** [comment]
This is a duplicate of #2059
